### PR TITLE
Added KIP13 to KIP7 for interfaceId

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces implementation KIP13 for KIP7.

When this PR merges, i'll add the supportsInterface function to Klaytn Docs.
Hopefully this PR will merge before the caver-js v1.4.0-rc.1 version is released.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #220 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @dev Interface of the KIP13 standard, as defined in the
+ * [KIP](https://klaytn.github.io/kips/KIPs/kip-interface_query_standard).
+ *
+ * Implementers can declare support of contract interfaces, which can then be
+ * queried by others (`KIP13Checker`).
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol]: Where can we find KIP13Checker?

Comment (jimni1222) [reply packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol]: I think the KIP13Checker in this description means supportsInterface function.

Comment (kjhman21) [reply packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol]: Do we have any definition of this somewhere?

Comment (jimni1222) [reply packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol]: There seems to be no definition for KIP13Checker.
I will remove ambiguous explanations.

**Pull request merged**